src/Component/Filter/ComparisonFilter/ComparisonFilter.tsx

```diff
@@ -21,19 +25,41 @@ import {
 
 import {
   get as _get,
-  cloneDeep as _cloneDeep
+  cloneDeep as _cloneDeep,
+  isEqual as _isEqual,
+  isNaN as _isNaN,
+  isNumber as _isNumber,
+  isEmpty as _isEmpty,
+  isFunction as _isFunction
 } from 'lodash';
 
 // default props
 interface DefaultComparisonFilterProps {
   filter: GsComparisonFilter;
   attributeNameFilter: (attributeName: string) => boolean;
+  attributeLabel: string | undefined;
```

Comment:
Not specifically realted to this PR/Change but i wonder if we couldn't make this optional instead of allowing `undefined`…

I for myself would prefer this syntax `attributeLabel?: string;` over `attributeLabel: string | undefined;`
But i don't know if this works…

Reply:
I'll check this

Reply:
Changed in 0158855